src/components/collection-card/components/with-auth-elements/index.tsx

```diff
@@ -0,0 +1,138 @@
+import { useMemo, ReactElement } from 'react'
+import { useCollectionProgress } from 'hooks/progress'
+import { parseCollectionProgress } from 'components/collection-progress-group'
+import ProgressBar from 'components/progress-bar'
+import {
+	CompleteIconAndLabel,
+	CountIconAndLabel,
+	CountIcon,
+	getStatusLabel,
+	StatusSectionElements,
+} from 'components/collection-progress-group/components'
+import CollectionCard, {
+	CollectionCardPropsWithId,
+} from 'components/collection-card'
+import s from './with-auth-elements.module.css'
+
+/**
+ * Displays a collection card, with progress elements where possible.
+ *
+ * Works without authentication enabled, displaying a tutorial count
+ * similar to the tutorial count displayed on plain collection cards.
+ *
+ * With authentication enabled, for authenticated users with progress,
+ * we display a progress bar or "complete" at the top of the card.
+ */
+function CollectionCardWithAuthElements({
```

Comment:
I've refactored this slightly to expand the layers of composition, as `CollectionCard` needs the same progress elements, but without the border and padding used in the sidebar and in the collection meta sections.

And, in the `"Complete"` state, there's a subtle difference - the `collection` icon is still shown, and the `Complete` text label is not visible.